isites_migration/views.py

```diff
@@ -29,7 +29,11 @@ def index(request):
         )
         return redirect('isites_migration:index')
 
-    processes = Process.objects.filter(name='isites_migration.jobs.migrate_files').order_by('-date_created')
+    processes = Process.objects.filter(
+        name='isites_migration.jobs.migrate_files',
+        details__at_canvas_course_id=canvas_course_id
```

Comment:
details['canvas_course_id'] is set in Process.enqueue()

https://github.com/Harvard-University-iCommons/django-icommons-common/blob/master/async/models.py#L74